src/components/CoverVideo/CoverVideo.scss

```diff
@@ -0,0 +1,7 @@
+.video-player {
+  height: 100% !important;
```

Comment:
Use tailwind.

Reply:
I don't think I can because this needs to be `important` to get over some dumb cover video stuff.

Reply:
Fair enough